app/uk/gov/hmrc/mobileusercontact/controllers/CSATFeedbackController.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mobileusercontact.controllers
+
+import play.api.libs.json.JsValue
+import play.api.mvc.{Action, ControllerComponents}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.mobileusercontact.domain.CSATFeedback
+import uk.gov.hmrc.mobileusercontact.domain.types.ModelTypes.JourneyId
+import uk.gov.hmrc.mobileusercontact.services.CSATFeedbackService
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+import uk.gov.hmrc.play.http.HeaderCarrierConverter.fromRequest
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+
+class CSATFeedbackController @Inject()(cc: ControllerComponents, service: CSATFeedbackService)(implicit ex: ExecutionContext)
+  extends BackendController(cc){
+
+  def feedback(journeyId: JourneyId) : Action[JsValue] =
+    Action.async(parse.json) { implicit request =>
```

Comment:
We should add authorisation to this endpoint now it is being called from the app directly